sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/SormasToSormasEncryptionEjb.java

```diff
@@ -44,25 +51,30 @@
 import de.symeda.sormas.api.SormasToSormasConfig;
 import de.symeda.sormas.api.i18n.I18nProperties;
 import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.api.sormastosormas.SormasToSormasApiConstants;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasEncryptedDataDto;
+import de.symeda.sormas.api.sormastosormas.SormasToSormasEncryptionFacade;
 import de.symeda.sormas.api.sormastosormas.SormasToSormasException;
 import de.symeda.sormas.backend.crypt.CmsCreator;
 import de.symeda.sormas.backend.crypt.CmsReader;
 
-@Stateless
-@LocalBean
-public class SormasToSormasEncryptionService {
+@Stateless(name = "SormasToSormasEncryptionFacade")
+public class SormasToSormasEncryptionEjb implements SormasToSormasEncryptionFacade {
```

Comment:
To follow the naming of other facades it should be SormasToSormasEncryptionFacadeEjb